lib/opensusebasetest.pm

```diff
@@ -49,7 +49,7 @@ sub clear_and_verify_console {
     my ($self) = @_;
 
     clear_console;
-    assert_screen('cleared-console');
+    assert_screen('cleared-console') unless is_serial_terminal();
```

Comment:
what do you do instead on the serial terminal? If you don't have this synchronisation points then I am sure followup test calls will fail. Otherwise the complete method would be useless, isn't it? Or in other words: Probably `clear_and_verify_console` really doesn't make any sense on serial terminals, does it?